README.rst

```diff
@@ -254,6 +254,71 @@ The ``DepositBox`` overrides the record's lifecycle management box on the deposi
 It takes care of rendering the "publish" button only when appropriate in the curation workflow.
 The other ``curationComponentOverrides`` provide better rendering for the new elements (e.g. event types) in the request page.
 
+Set Curation request custom field
+^^^^^^^^^^^^^^^^^^^^^^^^^
+Because there is a need to inform the user about the curation workflow, a custom field at the end is put just to show a specific message. In order to set this up, a basic invenio custom field in your instance should be configured.
+
+Create a new javascript file at assets/templates/custom_fields/RdmCuration.js
+
+.. code-block:: javascript
+
+    import React from 'react';
+
+    const RdmCuration = () => {
+      return (
+        <div className='ui visible warning message'>
+          <h4> Please create a curation request after saving the
+             draft by clicking on Start Publication Process
```

Comment:
nit: even it is documentation please mark them as translatable strings. that would add awareness that those messages should be translatable
and please even it is documentation indent it properly